pytezos-tests/pytezos_contract_tester.py

```diff
@@ -0,0 +1,589 @@
+        )["op_info"]
+    feetracker.add_result('addMerchFunding', op_info) 
+
+
+def test_reclaim():
+    print_header("Scenario test_reclaim: origination -> add_customer_funding -> reclaim_funding")
+
+    # Split the initial funding amounts to test a dual funded channel.
+    total_funding = cust_funding + merch_funding
+    merch_dual_funding = total_funding//2
+    cust_dual_funding = total_funding - merch_dual_funding
+
+    origination_op = originate(uri,
+        close_scalar_bytes,
+        cust_addr, merch_addr,
+        cust_acc,
+        cust_pubkey, merch_pubkey,
+        channel_id,
+        merch_g2, merch_y2s, merch_x2,
+        cust_dual_funding, merch_dual_funding,
+        min_confirmations, 
+        self_delay
+        )
+
+    add_customer_funding(
+        cust_acc,
+        origination_op["contract_id"],
+        cust_dual_funding,
+        min_confirmations
+        )
+
+    op_info = reclaim_funding(
+        cust_acc,
+        origination_op["contract_id"],
+        min_confirmations
+        )["op_info"]
+    feetracker.add_result('addMerchFunding', op_info) 
+
+# Define paths for all the files that will be used in the tests.
+establish_file = "sample_files/out.5f0b6efabc46808589acc4ffcfa9e9c8412cc097e45d523463da557d2c675c67.establish.json"
+close_file = "sample_files/out.5f0b6efabc46808589acc4ffcfa9e9c8412cc097e45d523463da557d2c675c67.close.json"
+tezos_account1 = "sample_files/tz1S6eSPZVQzHyPF2bRKhSKZhDZZSikB3e51.json"
+tezos_account2 = "sample_files/tz1VcYZwxQoyxfjhpNiRkdCUe5rzs53LMev6.json"
+zkchannel_contract = "../zkchannels-contract/zkchannel_contract.tz"
+
+# Load establish parameters from establish.json
+establish_json = read_json_file(establish_file)
+
+# Load variables from establish_json
+merch_ps_pk = establish_json.get("merchant_ps_public_key")
+close_scalar_bytes = establish_json.get("close_scalar_bytes")
+channel_id = establish_json.get("channel_id")
+cust_funding = establish_json.get("customer_deposit")
+merch_funding = establish_json.get("merchant_deposit")
+# Merchant's PS pubkey
+merch_g2 = merch_ps_pk.get("g2")
+merch_y2s = merch_ps_pk.get("y2s")
+merch_x2 = merch_ps_pk.get("x2")
+
+# Load variables from close_json
+cust_close_json = read_json_file(close_file)
+cs = cust_close_json.get("closing_signature")
+sigma1, sigma2 = cs.get("sigma1"), cs.get("sigma2")
+revocation_lock = cust_close_json.get("revocation_lock")
+customer_balance = cust_close_json.get("customer_balance")
+merchant_balance = cust_close_json.get("merchant_balance")
+
+# Load tezos account variables
+cust_acc = tezos_account1
+cust_addr = read_json_file(cust_acc)["pkh"]
+merch_acc = tezos_account2
+merch_addr = read_json_file(merch_acc)["pkh"]
+
+# Load the zkchannels contract
+zkchannel_contract = "../zkchannels-contract/zkchannel_contract.tz"
+main_code = ContractInterface.from_file(zkchannel_contract)
+
+# Query the blockchain to get the merchant's and customer's public keys
+uri = "https://rpc.tzkt.io/edo2net/"
```

Comment:
`uri` should be a command-line argument to provide flexibility for sandbox vs testnet